### PR TITLE
[AIFBR 6455] Added OOB metadata for Computer vision

### PR DESCRIPTION
**Pull request by SahilChaudhary25**

### CODE REVIEW

**Review by vivekjhauipath — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.iml
```
Comment (adeepsri) [inline .gitignore:1]: is this intended?

Comment (SahilChaudhary25) [reply .gitignore:1]: Yes

**Review by adeepsri — APPROVED** [review]
LGTM

**Pull request merged**